panel/src/api/files.js

```diff
@@ -9,7 +9,22 @@ export default (api) => {
 			return api.delete(parent + "/files/" + filename);
 		},
 		async get(parent, filename, query) {
-			let file = await api.get(parent + "/files/" + filename, query);
+			let file;
+
+			if (parent.includes("://") === true) {
+				// UUID: $api.files.get("file://hbjkh324k234", { query })
+				if (!query && filename) {
+					query = filename;
+				}
+
+				file = await api.get(
+					"files/@/" + parent.substring(parent.indexOf("://") + 3),
```

Comment:
I feel like the method should validate that the scheme is `file://` and then use the fixed offset of `7` for the `substring`.

This catches invalid uses such as `$api.files.get("page://hbjkh324k234")`. A call like this could happen by accident and it would be much easier to debug if there was a clear error in this case, even before the request is sent.